Dockerfile

```diff
@@ -1,12 +1,9 @@
-FROM node:14.15.4-buster
+FROM node:14.15.4-alpine
+ENV USER node
+ENV USER_ID 1000
```

Comment:
I don't think this line is used anymore?

Reply:
It's used on line 14.